text/0000-atcompoundkey-error-path-optimization.md

```diff
@@ -0,0 +1,197 @@
+</div>
+### Manifest file used for gathering the above metrics.
+```
+{
+		"relativelylongkeyname_element": true,
+		"relativelylongkeyname_map": {
+			"relativelylongkeyname_element": true,
+			"relativelylongkeyname_map": {
+				"relativelylongkeyname_element": true,
+				"relativelylongkeyname_map": {
+					"relativelylongkeyname_element": true,
+					"relativelylongkeyname_map": {
+						"relativelylongkeyname_element": true,
+						"relativelylongkeyname_map": {
+							"relativelylongkeyname_element": true,
+							"relativelylongkeyname_map": {
+								"relativelylongkeyname_element": true,
+								"relativelylongkeyname_map": {
+									"relativelylongkeyname_element": true,
+									"relativelylongkeyname_map": {
+										"relativelylongkeyname_element": true,
+										"relativelylongkeyname_map": {
+											"relativelylongkeyname_element": true,
+											"relativelylongkeyname_map": {
+												"relativelylongkeyname_element": true,
+												"relativelylongkeyname_map": {
+													"relativelylongkeyname_element": true,
+													"relativelylongkeyname_map": {
+														"relativelylongkeyname_element": true,
+														"relativelylongkeyname_map": {
+															"relativelylongkeyname_element": true,
+															"relativelylongkeyname_map": {
+																"relativelylongkeyname_element": true,
+																"relativelylongkeyname_map": {
+																	"relativelylongkeyname_element": true,
+																	"relativelylongkeyname_map": {
+																		"relativelylongkeyname_element": true,
+																		"relativelylongkeyname_map": {
+																			"relativelylongkeyname_element": true,
+																			"relativelylongkeyname_map": {
+																				"relativelylongkeyname_element": true,
+																				"relativelylongkeyname_map": {
+																					"relativelylongkeyname_element": true,
+																					"relativelylongkeyname_map": {
+																						"relativelylongkeyname_element": true,
+																						"relativelylongkeyname_map": {
+																							"relativelylongkeyname_element": true,
+																							"relativelylongkeyname_map": {
+																								"relativelylongkeyname_element": true,
+																								"relativelylongkeyname_map": {
+																									"relativelylongkeyname_element": true,
+																									"relativelylongkeyname_map": {}
+																								}
+																							}
+																						}
+																					}
+																				}
+																			}
+																		}
+																	}
+																}
+															}
+														}
+													}
+												}
+											}
+										}
+									}
+								}
+							}
+						}
+					}
+				}
+			}
+		}
+	}
+}
+```
+
+## Unresolved questions
+
+> Will we ever support null values in bundle's manifest.json file?
```

Comment:
If CppMicroServices wants to conform to the JSON standard, then eventually it should support ```null```

Reply:
I agree, users would expect `null` values to be supported if it is valid JSON syntax.